### PR TITLE
Huszarbalint development

### PR DESCRIPTION
**Pull request by balinthuszar**
new XML mode implementation finished

### CODE REVIEW

File: v1.0.1/template/Makefile [hunk 1]
```diff
@@ -17,23 +17,28 @@ LDFLAGS:=-g
...
 #----------------------------------------------------
 # Files
 #----------------------------------------------------
 # Name of target file
-TARGET:=Testsuite.exe
+TARGET:=Testsuite
 
 # Sourcefiles
 SRCS:=           \
-./System.c       \
```
Comment (balazsgrill) [inline v1.0.1/template/Makefile:30]: why was `System.c` removed?

**Review by balazsgrill — COMMENTED** [review]
@StackedWeeb 
I don't see where System.c is compiled in the makefile

File: v1.0.1/template/Makefile (hunk 1 above)
Comment (balinthuszar) [reply v1.0.1/template/Makefile:30]: good question. could not find a reason for that, it was a mistake

**Review by balinthuszar — COMMENTED** [review]
im correcting the makefile right away

**Review by balazsgrill — APPROVED** [review]

**Pull request merged**